### ISSUE TITLE
TF-Slim Walkthrough slim/slim_walkthough.ipynb Doesn't Work With TF 0.12

### ISSUE THREAD

**Issue by dweekly** [tensorflow/models#782]
The walkthrough has the user use the sum_of_squares() loss function which was removed after TF 0.10 and is no longer present in TF 0.12. Please update the walkthrough to work with TF 0.12.

loss = slim.losses.sum_of_squares(predictions, targets)

**Comment by drstarson** [comment]
The suggested solution worked for me.
Update sum_of_squares (TF 0.10) to mean_squared_error  (TF 0.12)

**Comment by nealwu** [comment]
https://github.com/tensorflow/models/pull/783 has been merged.
